content/blog/dplyr-1-1-0-is-coming-soon/index.Rmd

```diff
@@ -0,0 +1,548 @@
+```
+
+We are optimistic that this change is an overall net positive.
+We anticipate that many users use `arrange()` to simply group similar looking observations together, and we expect that the main places you'll need to care about localized ordering are the few places when you are generating human readable output, such as a table or a chart, at which point you might consider using `.locale`.
+
+If you are having trouble converting an existing script over to the new behavior, you can set the temporary global option `options(dplyr.legacy_locale = TRUE)`, which will revert to the pre-1.1.0 behavior of using the system locale.
+We expect to remove this option in a future release.
+
+To learn more low-level details about this change, you can read our [tidyup](https://github.com/tidyverse/tidyups/blob/main/003-dplyr-radix-ordering.md).
+
+## `morph()`, a generalization of `summarise()`
+
+In dplyr 1.0.0, we introduced a powerful new feature: `summarise()` could return per-group results of any length, rather than just length 1.
+For example:
+
+```{r, include=FALSE}
+options(lifecycle_verbosity = "quiet")
+```
+
+```{r}
+table <- c("a", "b", "d", "f")
+
+df <- tibble(
+  g = c(1, 1, 1, 2, 2, 2, 2),
+  x = c("e", "a", "b", "c", "f", "d", "a")
+)
+
+df %>%
+  summarise(x = intersect(x, table), .by = g)
+```
+
+```{r, include=FALSE}
+options(lifecycle_verbosity = NULL)
+```
+
+While extremely powerful, community feedback has raised the valid concern that allowing `summarise()` to return any number of rows per group:
+
+-   Increases the chance for accidental bugs
+
+-   Is against the spirit of a "summary," which implies 1 row per group
+
+-   Makes translation to dbplyr very difficult
+
+We agree!
+In response to this, we've decided to walk back that change to `summarise()`, which will now throw a warning when either 0 or \>1 rows are returned per group:
+
+```{r, include=FALSE}
+options(lifecycle_verbosity = "warning")
+```
+
+```{r}
+df %>%
+  summarise(x = intersect(x, table), .by = g)
+```
+
+```{r, include=FALSE}
+options(lifecycle_verbosity = NULL)
+```
+
+That said, we still believe that this is a powerful tool, so we've moved these features to a new verb, `morph()`.
+Think of `morph()` as a generic tool for "doing something to each group," with no restrictions on the number of rows returned per group.
+
+```{r}
+df %>%
+  morph(x = intersect(x, table), .by = g)
+```
+
+One big difference between `summarise()` and `morph()` is that `morph()` always returns an ungrouped data frame, even if the input was a grouped data frame with multiple groups.
+This simplifies `morph()` immensely, as it doesn't need to inherit the `.groups` argument of `summarise()`, and never emits any messages.
+
+We expect that you'll continue to use `summarise()` much more often than `morph()`, but if you ever find yourself applying a function to each group that returns an arbitrary number of rows, `morph()` should be your go-to tool!
+
+## Vector function rewrites
```

Comment:
I'd be tempted to omit this since I'm not sure it needs comment. (You can still use it in the release blog post)

Reply:
Are you talking about the little paragraph that follows this, or this whole section? Did you not want to talk about `case_when()` in the pre-release post? I intended for this to be an H2 header to set up the three H3 headers about `case_when` `case_match` and `consecutive_id`

Reply:
I was thinking the whole section.

Reply:
I removed it